src/sbt-test/sbt-cake/simple/src/main/scala/Hello.scala

```diff
@@ -1 +1 @@
-object TerribleFormatting { def foo = "bar" }
+object TerribleFormatting{def foo=   "bar"}
```

Comment:
if you feel that people need a comment on a badly formatted piece of code, literally called `TerribleFormatting`, then by all means go ahead and add one.